packages/hash/api/src/model/ontology/dataType.model.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  DataType as DataTypeSchema,
```

Comment:
I personally think these renames are dangerous and confusing and we should instead just pick more descriptive terms 🙃 

I appreciate this is somewhat different to how we've been naming our model classes though so it's just a suggestion

Reply:
Well the first thing I'd say is that a Data Type _is_ a schema, it's not anything else, so it seems a bit strange to rename it here.

Then we seem to shove it inside something else that on the Graph side is more aligned with a "qualified" X. But I think we're also shadowing `DataType` as being the model class, which I also don't particularly like but I can understand why it's been done at least.

So just some immediate thoughts that aren't necessarily well-reasoned:
* `DataType`
* `QualifiedDataType`
* `DataTypeModel`

Reply:
Addressed in 9937ccb617187c2dc091f04d0977995d90318b3c